src/CppAst.Tests/CppAst.Tests.csproj

```
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
I think we can move to 3.1 directly

---

Same can be said of all projects?

---

Executables yes, not libraries.